hack/generate-operator-bundle-contents.py

```diff
@@ -1,35 +1,47 @@
 #!/usr/bin/env python3
 
+# vim: ts=4:sw=4:cc=99
+
 import datetime
 import os
 import sys
 import yaml
 import pathlib
 import argparse
 
-parser = argparse.ArgumentParser()
-parser.add_argument("-r", "--replaces", help="Replaces version", type=str)
-parser.add_argument('-s','--skip', action='append',
+parser = argparse.ArgumentParser(add_help=False)
+required = parser.add_argument_group('required arguments')
+
+required.add_argument('-n', '--name', help='operator name', type=str, required=True)
+required.add_argument('-v', '--version', help='operator version', type=str, required=True)
```

Comment:
nit: `--version` makes me think I'm going to get the version of the tool.
Since we're dealing with multiple "version" args (`replaces`, `skip`), perhaps this could be `-c`/`--current-version`?

This is totally not a big deal though.

Reply:
I like the idea. I will change it.

Reply:
fixed in 7415dc6